specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2383 @@
+    },
+    "AnalyzeConversationLROTaskKind": {
+      "type": "string",
+      "description": "Enumeration of supported analysis tasks on a collection of conversations.",
+      "enum": [
+        "ConversationalPIITask",
+        "ConversationalSummarizationTask"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationLROTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported Conversation Analysis task results.",
+      "enum": [
+        "ConversationalPIIResults",
+        "ConversationalSummarizationResults"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationPIITask": {
+      "type": "object",
+      "description": "Task definition for a PII redaction in conversations.",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/ConversationPIITaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "ConversationalPIITask"
+    },
+    "ConversationPIIItemResult": {
+      "type": "object",
+      "required": [
+        "id",
+        "redactedContent",
+        "entities"
+      ],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "redactedContent": {
+          "type": "object",
+          "$ref": "#/definitions/RedactedTranscriptContent"
+        },
+        "entities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Entity"
+          }
+        }
+      }
+    },
+    "MultiLanguageConversationAnalysisInput": {
+      "required": [
+        "conversations"
+      ],
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Conversation"
+          }
+        }
+      }
+    },
+    "Conversation": {
+      "type": "object",
+      "description": "A complete ordered set of utterances (spoken or written), by one or more speakers to be used for analysis.",
+      "discriminator": "modality",
+      "required": [
+        "id",
+        "language",
+        "modality"
+      ],
+      "properties": {
+        "id": {
+          "description": "Unique identifier for the conversation.",
+          "type": "string"
+        },
+        "language": {
+          "description": "The language of the conversation item in BCP-47 format.",
+          "type": "string"
+        },
+        "modality": {
+          "$ref": "#/definitions/Modality"
+        },
+        "domain": {
+          "$ref": "#/definitions/ConversationDomain"
+        }
+      }
+    },
+    "Modality": {
+      "type": "string",
+      "description": "Enumeration of supported conversational modalities.",
+      "enum": [
+        "transcript",
```

Comment:
Something to consider (at least before going GA with the conversational API) is to align enum casing to be consistent, if possible.

In the TA spec we mostly use Pascal case, but do have some inconsistencies in our naming which generally invites a lot of criticism when reviewers notice it.  :)

Reply:
agreed will check

Reply:
Updated to Pascal case to match existing eums.